kill.go

```diff
@@ -129,7 +130,27 @@ func kill(containerID, signal string, all bool) error {
 		return fmt.Errorf("Process not running inside container %s", containerID)
 	}
 
-	if err := vc.KillContainer(containerID, podStatus.ContainersStatus[0].ID, signals[signal]); err != nil {
+	signum, signalOk := signals[signal]
```

Comment:
You have the same code repeated twice now with some somewhat deep nested indentation.

```
				if err := vc.KillContainer(containerID, podStatus.ContainersStatus[0].ID, signum); err != nil {
					return err
				}
				return nil
```

1. resolve the argument to get a signal number
2. do the KillContainer call

For 1. we don't actually have to cycle through the map, we can just test if the signal is between 1 and 31